packages/elements/src/radio-button/index.ts

```diff
@@ -23,7 +23,7 @@ import {
 /**
  * Basic radio button
  *
- * @fires checked-changed - Fired when the `checked` property changes.
+ * @fires checked-changed - Fired when the user checked or unchecked the radio. The event is not fired if the `checked` property is changed by programmatically.
```

Comment:
Fired when the user checks~~ed~~ or unchecks~~ed~~ the radio. The event is not fired if the `checked` property is changed ~~by~~ programmatically.